src/sentry/static/sentry/app/stores/guideStore.jsx

```diff
@@ -99,7 +115,11 @@ const GuideStore = Reflux.createStore({
     }
 
     this.state.currentGuide = bestGuide;
-    this.state.currentStep = 0;
+
+    if (forceShow && this.state.currentGuide) {
+      this.state.currentStep = this.state.currentStep || 1;
```

Comment:
Not sure I understand why we need something other than `currentStep = forceShow ? 1 : 0` here?

Reply:
fixed